### DISCUSSION TITLE
[firebase_ml_vision] Add rawBytes to the Barcode class

### DISCUSSION THREAD

**Discussion by the-mars-rover** [firebase/flutterfire#5582] [Feature request]
The problem I am facing:
I need to scan barcodes which are in ISO 8859-1 (latin1) format. However, the `rawValue` property of the `Barcode` class has already been encoded using UTF-8 and contains some replacement characters (�). Consequently, I am unable to determine what the actual value of the data was.

Proposed Solution:
Add a `rawBytes` property to the `Barcode` class - populate this property using [`getRawBytes`](https://firebase.google.com/docs/reference/android/com/google/firebase/ml/vision/barcode/FirebaseVisionBarcode.html#getRawBytes()) on Android and [`rawData`](https://firebase.google.com/docs/reference/ios/firebasemlvision/api/reference/Classes/FIRVisionBarcode#/c:objc(cs)FIRVisionBarcode(py)rawData) on iOS. Then, in cases that the barcode follows formats other than UTF-8, `rawBytes` can be used with the applicable format.

Some problems which are similar to this (even though they are on different projects):
- https://github.com/rmtmckenzie/flutter_qr_mobile_vision/issues/44#issuecomment-426445584
- https://github.com/googlesamples/android-vision/issues/156

If there is a better solution to this problem I really would like some help.

**Comment by the-mars-rover** [comment]
Submitted a PR for this: https://github.com/FirebaseExtended/flutterfire/pull/2368

**Comment by the-mars-rover** [comment]
Updated the linked PR by pulling in latest master

**Comment by dayron9110** [comment]
Any updates?

**Comment by dayron9110** [comment]
News?

**Comment by the-mars-rover** [comment]
Since the ```firebase_ml_vision``` plugin was discontinued, followers of this issue can be directed to https://github.com/bharat-biradar/Google-Ml-Kit-plugin/pull/56
